wine-build/Dockerfile

```diff
@@ -1,13 +1,7 @@
-FROM i386/alpine:3.12
-
-# Install prerequisites
-RUN apk add libnitrokey git wget build-base \
-				mingw-w64-binutils mingw-w64-gcc \
-				make autoconf automake ncurses freetype libtool \
-				gcc g++ wine gnutls p7zip
-
+FROM tobix/pywine:3.9
```

Comment:
I gave this one brief review. Looks good. 
Perhaps worth leaving a URL to the project somewhere (docs or Dockerfile):
- https://github.com/webcomics/pywine